packages/plugins/test/Validation.spec.ts

```diff
@@ -63,23 +63,31 @@ describe('Validate', () => {
       process.stderr
     )
 
-    await readValidatedChugSplashConfig(
-      provider,
-      variableValidateConfigPath,
-      varValidationArtifactPaths,
-      'hardhat',
-      cre,
-      false
-    )
+    try {
+      await readValidatedChugSplashConfig(
+        provider,
+        variableValidateConfigPath,
+        varValidationArtifactPaths,
+        'hardhat',
+        cre,
+        false
+      )
+    } catch (e) {
+      /* empty */
+    }
 
-    await readValidatedChugSplashConfig(
-      provider,
-      constructorArgConfigPath,
-      constructorArgsValidationArtifactPaths,
-      'hardhat',
-      cre,
-      false
-    )
+    try {
+      await readValidatedChugSplashConfig(
+        provider,
+        constructorArgConfigPath,
+        constructorArgsValidationArtifactPaths,
+        'hardhat',
+        cre,
+        false
+      )
+    } catch (e) {
```

Comment:
Needed to add these try/catch statements because I added a call to `getCreationCodeWithConstructorArgs` in `constructParsedConfig`, which throws an error when there's incorrectly encoded constructor args.